src/shipit_code_coverage_backend/shipit_code_coverage_backend/v2/__init__.py

```diff
@@ -42,3 +44,41 @@ def coverage_for_path(path='', changeset=None):
         'type': 'directory',
         'children': paths
     }
+
+
+def coverage_for_revision(changeset):
+    '''
+    List coverage changes for any revision
+    '''
+    assert active_data.enabled, \
+        'Only ActiveData is supported'
+
+    # Run slow tasks in parallel
```

Comment:
The coverage frontend requires the patch data too, the addon doesn't.
Maybe we can add an option to retrieve the raw patch too? Or maybe we should have two separate endpoints, one retrieving both and one retrieving the coverage data only?